marketplace/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Builder
+FROM golang:1.15-alpine as builder
+
+WORKDIR /go/src/github.com/grafeas/voucher
+COPY . .
+RUN apk --no-cache add \
+    git \
+    make && \
+    make voucher_server
+
+# Final build
+FROM alpine:3.8
```

Comment:
How does this interact with line 2?  I didn't even know you could 'FROM' two different images.

Reply:
The first base image is used simply for building the voucher_server, and then everything is left out except for the voucher_server which is explicitly copied to the second base image.